nodejs/sqlcommenter-nodejs/packages/sqlcommenter-sequelize/provider/index.js

```diff
@@ -0,0 +1,19 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const {OpenCensusProvider} = require('./opencensusprovider');
+const {OpenTelemetryProvider} = require('./opentelemtryprovider');
+
+exports.OpenCensusProvider = OpenCensusProvider;
+exports.OpenTelemetryProvider = OpenTelemetryProvider;
```

Comment:
I don't think we need to export OpenCensusProvider nor OpenTelemetryProvider, we can just create the logic in here
to multiplex on the various providers thus
```javascript
function attachComments(providerName, comments) {
     // Lookup the provider by name, or use the default.
     provider = providers[providerName] || OpenCensus;
}
```